src/highlightui.js

```diff
@@ -10,13 +10,63 @@
 import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
 
 import HighlightEditing from './highlightediting';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+
+import markerIcon from './../theme/icons/marker.svg';
+import penIcon from './../theme/icons/pen.svg';
+import eraserIcon from './../theme/icons/eraser.svg';
+
+import ToolbarSeparatorView from '@ckeditor/ckeditor5-ui/src/toolbar/toolbarseparatorview';
+import SplitButtonView from '@ckeditor/ckeditor5-ui/src/dropdown/button/splitbuttonview';
+import { createDropdown, addToolbarToDropdown } from '@ckeditor/ckeditor5-ui/src/dropdown/utils';
+
+import './../theme/highlight.css';
 
 /**
- * The default Highlight UI plugin.
+ * The default Highlight UI plugin. It introduces:
+ * * the `'highlightDropdown'` drop-down,
+ * * `'removeHighlight'` and `'highlight:*'` buttons.
+ *
+ * The default configuration includes the following buttons:
+ * * `'highlight:marker'`
+ * * `'highlight:pinkMarker'`
+ * * `'highlight:greenMarker'`
+ * * `'highlight:redPen'`
+ * * `'highlight:bluePen'`
+ *
+ * See the {@link module:highlight/highlight~HighlightConfig#options configuration} to learn more
+ * about the defaults.
  *
  * @extends module:core/plugin~Plugin
  */
 export default class HighlightUI extends Plugin {
+	/**
+	 * Returns the localized option titles provided by the plugin.
+	 *
+	 * The following localized titles corresponding with default
+	 * {@link module:highlight/highlight~HighlightConfig#options} are available:
+	 *
+	 * * `'Marker'`,
+	 * * `'Green marker'`,
+	 * * `'Pink marker'`,
+	 * * `'Blue pen'`.
+	 * * `'Red pen'`.
+	 *
+	 * @readonly
+	 * @type {Object.<String,String>}
+	 */
+	get localizedOptionTitles() {
+		const t = this.editor.t;
+
+		return {
+			'Marker': t( 'Marker' ),
```

Comment:
Why is the yellow marker just "Marker" while the green one is "Green marker"? This is inconsistent and potentially confusing for the user with visual impairments.